### ISSUE TITLE
Russian localization for Pimatic (Русская локализация)

### ISSUE THREAD

**Issue by mocart2** [pimatic/pimatic#553]
Hi everyone, my wife is prepare Russian translation for en.json localization file: if you want, you can use it: (Если будут замечания, прошу отправлять их сюда)
```
{
    "on": "вкл",
    "turned %s on": "включено %s",
    "Please enter a id": "Введите ИД",
    "would turn %s on": "перключить %s вкл",
    "would log \"%s\"": "журнал \"%s\"",
    "turned %s off": "выключено %s",
    "log: \"%s\"": "Журнал: \"%s\"",
    "Off": "Выкл",
    "On": "Вкл",
    "Menu": "Меню",
    "Speak": "Голосовой ввод",
    "Devices": "Устройства",
    "Drop here to remove item": "Перетащите сюда, чтобы удалить пункт",
    "Rules": "Правила",
    "Close menu": "Закрыть меню",
    "Log": "Журнал",
    "Back": "Назад",
    "Add a item": "Добавить пункт",
    "Actuators": "Актуаторы",
    "Sensors": "Сенсоры",
    "if": "если",
    "then": "то",
    "Edit rule": "Редактировать правило",
    "Add new rule": "Добавить новое правило",
    "Cancel": "Отмена",
    "Save": "Сохранить",
    "Delete this rule": "Удалить это правило",
    "Clear": "Очистить",
    "Log messages": "Сообщения из журнала",
    "would turn %s off": "выключено %s",
    "Add an item": "Добавить пункт",
    "Add a new rule": "Добавить новое правило",
    "Advanced Options": "Дополнительные настройки",
    "connection lost, retying": "соединение потеряно, повтор",
    "The rule must start with \"if\" and contain a \"then\" part!": "Правило должно начинаться с \"if\" и содержать \"then\"!",
    "No id given": "Не получено id",
    "Plugins": "Плагины",
    "Installed Plugins": "Установленные плагины",
    "Install new Plugin": "Установить новый плагин",
    "Do Action": "Вперед!",
    "select a action": "Выбрать действие",
    "add to config": "добавить в конфигурацию",
    "remove from config": "удалить из конфигурации",
    "do it": "вперед",
    "Select a action": "Выбрать действие",
    "Add to config": "Добавить в конфигурацию",
    "Remove from config": "Удалить из конфигурации",
    "Do it!": "Вперед!",
    "Install a new Plugin": "Установить новый плагин",
    "Install a Plugin": "Установка плагина",
    "Select an action": "Выбрать действие",
    "Install a Plugins": "Установка плагинов",
    "Add": "Добавить",
    "Loading": "Загрузка",
    "Updates": "Обновления",
    "Install Updates": "Установка обновлений",
    "active": "активный",
    "update available": "доступно обновление",
    "installed": "установлено",
    "Restart pimatic": "Перезапуск pimatic",
    "Add a group header": "Добавить название группы",
    "Other": "Другое",
    "Enabled": "Включено",
    "Sensor value log": "Журнал значений сенсора",
    "Yes": "Да",
    "No": "Нет",
    "Sensor values": "Значения сенсора",
    "Show": "Показать",
    "Graph": "График",
    "Log data: ": "Данные журнала: ",
    "Show data of: ": "Показать данные: ",
    "Last 24 hours": "За 24 часа",
    "Last 7 days": "За 7 дней",
    "Last 30 days": "За 30 дней",
    "Last 365 days": "За 365 дней",
    "Id": "Ид",
    "Name": "Название",
    "ID": "ИД",
    "Device info": "Информация об устройстве",
    "Attributes": "Атрибуты",
    "Add a custom button": "Добавить кнопку",
    "Added %s to the list.": "Добавлено %s в список.",
    "You can now edit your list.": "Теперь вы можете изменить свой список.",
    "Editing the list locked.": "Редактируемый список заблокирован.",
    "The list is now locked.": "Список заблокирован.",
    "would execute \"%s\"": "будет выполнено \"%s\"",
    "executed \"%s\": %s": "выполненео \"%s\": %s",
    "dimmed %s to %s%%": "приглушенный %s до %s%%",
    "would dim %s to %s%%": "будет приглушен %s до %s%%",
    "Remember me": "Запомнить меня",
    "Show CA root certificate": "Показать корневой сертификат CA",
    "Restart": "Перезапуск",
    "Last Update: ": "Последнее обновление: ",
    "Button": "Кнопка",
    "Add a button": "Добавить кнопку",
    "unknown": "неизвестный",
    "connecting": "Соединение",
    "Variables": "Переменные",
    "Choose an action": "Выберите действие",
    "Change order": "Изменить порядок",
    "Add a variable": "Добавить переменную",
    "Showing variables for device attributes.": "Отображение переменных для атрибутов устройства.",
    "Hiding variables for device attributes.": "Скрытие переменных для атрибутов устройства.",
    "Value": "Значение",
    "Delete this variable": "Удалить данную переменную",
    "Down": "Вниз",
    "Up": "Вверх",
    "Expression": "Значение",
    "Type": "Тип",
    "pusover message sent successfully": "сообщение успешно отправлено",
    "Drop here to remove rule": "Перетащите сюда, чтобы удалить правило",
    "Drop here to remove variable": "Перетащите сюда, чтобы удалить значение",
    "Description of the rule": "Описание правила",
    "Short Description": "Краткое Описание",
    "Short description": "Кратеое описание",
    "lowered %s down": "понижено %s",
    "lifted %s up": "повышено %s",
    "mail sent with statuscode: %s": "сообщение отправлено с кодовым статусом: %s",
    "mail sent with status: %s": "сообщение отправлено со статусом: %s",
    "raised %s": "повышено %s",
    "lowered %s": "понижено %s",
    "stopped %s": "остановлено %s",
    "There was an error installing the updates.": "Ошибка при установке обновлений.",
    "Updates were successful. Please restart pimatic.": "Обновления прошли успешно. Пожалуйста перезапустите Pimatic.",
    "Log ": "Журнал ",
    "Log action execution ": "Журнал выполнения действий ",
    "Log Action Execution ": "Журнал выполнения действий ",
    "Copy as new": "Копировать как новый",
    "Delete All": "Удалить все",
    "Events": "События",
    "Pages": "Страницы",
    "Messages": "Сообщения",
    "Settings": "Настройки",
    "Automation": "Автоматизация",
    "Delete this Page": "Удалить эту страницу",
    "Ungrouped": "Без группы",
    "Device Attributes": "Атрибуты устройства",
    "Add a new item": "Добавить новый пункт",
    "Time": "Время",
    "Device": "Устройство",
    "Attribute": "Атрибут",
    "Groupes": "Группы",
    "Drop here to remove group": "Перетащите сюда, чтобы удалить группу",
    "Add a new group": "Добавить новую группу",
    "Groups": "Группы",
    "Delete this group": "Удалить эту группу",
    "Show: ": "Показать: ",
    "Delete This Rule": "Удалить это правило",
    "Copy as New": "Копировать как новое",
    "Delete this Group": "Удалить эту группу",
    "Class": "Класс",
    "Drop here to remove device": "Перетащите сюда, чтобы удалить устройство",
    "Add a new device": "Добавить новое устройство",
    "Delete this Device": "Удалить это устройство",
    "Showen devices": "Отображаемые устройства",
    "Hide": "Скрыть",
    "pushover message sent successfully": "сообщение успешно отправлено",
    "Show data of ": "Показать данные за ",
    "24 hours": "24 часа",
    "7 days": "7 дней",
    "30 days": "30 дней",
    "365 days": "365 дней",
    " from ": " от ",
    "up to ": "до ",
    "Show ": "Показать ",
    "played \"%s\"": "воспроизведено \"%s\"",
    "toggled state of %s": "переключено состояние %s",
    "Edit Pages": "Редактировать страницы",
    "Drop here to remove page": "Перетащите сюда, чтобы удалить страницу",
    "Add a new page": "Добавить новую страницу",
    "Previous": "Предыдущий",
    "Next": "Следующий",
    "I'm an empty page, add some items!": "Пустая страница, добавьте пункты!",
    "Rpicam": "Rpicam",
    "Capture Image": "Захватить изображение",
    "There was an error installing the updates, please check the log.": "роизошла ошибка при установке обновлений, пожалуйста проверьте журнал.",
    "Permission denied": "Запрещено",
    "Logout": "Выход",
    "Login": "Войти",
    "Username": "Имя пользователя",
    "Password": "Пароль",
    "Wrong username or password": "Неправильное имя пользователя или пароль",
    "Wrong username or password.": "Неправильное имя пользователя или пароль.",
    "Keep logged in": "Запомнить меня",
    "Keep me logged in": "Оставаться в системе",
    "Link": "Ссылка",
    "Config": "Конфигурация",
    "Unit": "Блок",
    "Database": "База данных",
    "Problems": "Проблемы",
    "No problems found.": "Проблемы не обнаружены.",
    "Condition": "Состояние",
    "And": "И",
    "Or": "Или",
    "and": "и",
    "or": "или",
    "all": "все",
    "OK": "ОК",
    "Predicate": "Предикат",
    "Select Event": "Выбрать событие",
    "Available Events": "Доступные события",
    "Event": "Событие",
    "switch to text input": "переключить в текстовый режим",
    "switch to text input mode": "переключить в текстовый режим",
    "switch to text mode": "переключить в текстовый режим",
    "switch to gui mode": "переключить в графический режим",
    "Duration": "Продолжительность",
    "Attribute Info": "Информация об атрибуте",
    "Interval": "Интервал",
    "Expire": "Истекать",
    "Count": "Счетчик",
    "Attribute logging info": "Атрибуты служебной информации",
    "Remove": "Удалить",
    "Stop": "Стоп"
}
```

**Comment by sweetpi** [comment]
Hi, could you make a pull request for that by adding a `ru.json` file directly on github: https://help.github.com/articles/creating-new-files

**Comment by mocart2** [comment]
I sent request, but git write ti me:
```
This branch has failed checks, but can be merged.
Only those with write access to this repository can merge pull requests.
```

**Comment by sweetpi** [comment]
That's all fine. I will merge it with the next release. Thank you very much.

**Comment by sweetpi** [comment]
Merged, watch #557 for missing translations.
